### ISSUE TITLE
An error occurred while Style/TrailingBlankLines cop was inspecting

### ISSUE THREAD

**Issue by JaniJegoroff** [bbatsov/rubocop#1843]
Just noticed minor bug related to Style/TrailingBlankLines cop.

**Given .rb file content**

```ruby
puts 'testing rubocop when final new line is missing after block comments'

=begin
first line
second line
third line
=end
```

**Output:**
```
Janis-MacBook-Pro:rubo janijegoroff$ rubocop rubo.rb
warning: parser/current is loading parser/ruby21, which recognizes
warning: 2.1.6-compliant syntax, but you are running 2.1.2.
Inspecting 1 file
An error occurred while Style/TrailingBlankLines cop was inspecting /Users/janijegoroff/projects/rubo/rubo.rb.
To see the complete backtrace run rubocop -d.
C

Offenses:

rubo.rb:3:1: C: Do not use block comments.
=begin
^^^^^^

1 file inspected, 1 offense detected

1 error occurred:
An error occurred while Style/TrailingBlankLines cop was inspecting /Users/janijegoroff/projects/rubo/rubo.rb.
Errors are usually caused by RuboCop bugs.
Please, report your problems to RuboCop's issue tracker.
Mention the following information in the issue report:
0.30.1 (using Parser 2.2.2.1, running on ruby 2.1.2 x86_64-darwin13.0)
```
